t/pkg-Cpanel-Security-Advisor-Assessors-Imunify360.t

```diff
@@ -0,0 +1,103 @@
+#!/usr/local/cpanel/3rdparty/bin/perl
+
+# Copyright (c) 2018, cPanel, L.L.C.
+# All rights reserved.
+# http://cpanel.net
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#     * Redistributions of source code must retain the above copyright
+#       notice, this list of conditions and the following disclaimer.
+#     * Redistributions in binary form must reproduce the above copyright
+#       notice, this list of conditions and the following disclaimer in the
+#       documentation and/or other materials provided with the distribution.
+#     * Neither the name of the owner nor the names of its contributors may
+#       be used to endorse or promote products derived from this software
+#       without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL  BE LIABLE FOR ANY
+# DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+use strict;
+use warnings;
+
+use FindBin;
+use lib "$FindBin::Bin/lib", "$FindBin::Bin/../pkg";
+
+use Cpanel::Version ();
+use Test::Assessor  ();
+use Test::More;
+use Test::MockModule;
+
+use Cpanel::Security::Advisor::Assessors::Imunify360 ();
+
+plan skip_all => 'Requires cPanel & WHM v80 or later' if Cpanel::Version::compare( Cpanel::Version::getversionnumber(), '<', '11.79' );
+
+plan tests => 4;
+
+my $mocked_version_module    = Test::MockModule->new('Cpanel::Version');
+my $mocked_imunify360_module = Test::MockModule->new('Whostmgr::Imunify360');
+
+subtest 'Requires cPanel v80 or later' => sub {
+    plan tests => 1;
+
+    $mocked_version_module->mock( getversionnumber => sub { '11.70' } );
+
+    my $advice = get_advice();
+
+    ok( eq_array( \$advice, \[] ), "Should not get advice for versions lower than 80" ) or diag explain $advice;
```

Comment:
A couple suggestions here:

Instead of `ok( eq_array( ... ) )`, it would be better to use `is_deeply()`. Any time you use an `ok()`, the failure diagnostic output will be limited to an ok / not ok result (plus the diag you have here). Data structure comparison functions like `is_deeply()` can provide specific information about where the mismatch occurred and what it got vs expected in that position.

There is no need to add the `\` to turn these two things into references, as they are already both references.